experiments/modern/src/utils.js

```diff
@@ -139,3 +140,44 @@ export function findDescendantByTypeAndId(node, type, id) {
 
   return null;
 }
+
+function getPreviousSiblings(node, parent) {
+  const children = parent.children;
+  for (let i = 0; i < children.length; i++) {
+    if (children[i] === node) {
+      if (i === 0) {
+        return [];
+      }
```

Comment:
I don’t think we need this special case.

Reply:
yup! works without it